docs/book/src/bigint/index.md

```diff
@@ -0,0 +1,142 @@
+# Big Integers Library
+
+<!--Include BigInt when BigInt is available-->
+The Big Integers library provides a library to use extremely large numbers in Sway. It has 1 distinct types: `BigUint`. These types are heap allocated.
```

Comment:
```suggestion
The Big Integers library provides a library to use extremely large numbers in Sway. It has 1 distinct type: `BigUint`. This type is heap allocated.
```

Reply:
This is done in preparation for `BigInt`